nestedbox_type.admin.inc

```diff
@@ -78,3 +78,37 @@ function nestedbox_type_form_submit_delete(&$form, &$form_state) {
   $nestedbox_type = $form_state['nestedbox_type'];
   $form_state['redirect'] = 'admin/structure/nestedbox-types/manage/' . $nestedbox_type->type . '/delete';
 }
+
+/**
+ * Nestedbox type delete form.
+ */
+function nestedbox_type_form_delete_confirm($form, &$form_state, $task_type) {
```

Comment:
Wow, I notice this function was registered in hook_menu() all the time, but the function did not exist. So, good move to add this function!